website/pages/en/deploying/deploying-a-subgraph-to-studio.mdx

```diff
@@ -25,44 +44,103 @@ yarn global add @graphprotocol/graph-cli
 npm install -g @graphprotocol/graph-cli
 ```
 
-## Create your Subgraph in Subgraph Studio
+## Create your Subgraph
+
+### Create an Account
+
+Before deploying your subgraph you need to create a subgraph in [Subgraph Studio](https://thegraph.com/studio/).
+
+1. Open [Subgraph Studio](https://thegraph.com/studio/).
+2. Connect your wallet to sign in.
+   - You can do this via MetaMask, Coinbase Wallet, WalletConnect, or Safe.
+3. After you sign in, your unique deploy key will be displayed on your account home page.
+   - The deploy key allows you to publish your subgraphs or manage your API keys and billing. It is unique but can be regenerated if you think it has been compromised.
```

Comment:
That doesn't seem right. The deploy key is shown on subgraph details pages, not on the "account home page", whatever that is (the Settings page?). It _would be nice_ if it was shown on the Settings page, since it is account-specific and not subgraph-specific. 🤔 cc @IainM22